clients/go/README.md

```diff
@@ -1 +1,50 @@
-# Raccoon Client Library for Go
\ No newline at end of file
+# Raccoon Client Library for Go #
+
+[![Go Reference](https://pkg.go.dev/badge/google.golang.org/api.svg)](https://pkg.go.dev/google.golang.org/api)
+
+## Requirements
+go 1.16 or above
+
+## Install
+```shell
+go get github.com/odpf/raccoon/clients/go
+```
+## Usage
+
+#### Construct a new REST client, then use the various options on the client.
+For example:
+```go
+	client, err := rest.NewRest(
+		rest.WithUrl("http://localhost:8080/api/v1/events"),
+		rest.WithHeader("x-user-id", "123"),
+		rest.WithSerializer(serializer.PROTO), // default is JSON
+	)
+```
+#### Construct a new GRPC client, then use the various options on the client.
+For example:
+```go
+	client, err := grpc.NewGrpc(
```

Comment:
@AkbaraliShaikh let's show the import statement as well here since it is sub package imort.

Reply:
updated